.script/tests/KqlvalidationsTests/CustomTables/ZoomLogs.json

```diff
@@ -40,7 +40,26 @@
     {
       "Name": "payload_object_participant_user_name_s",
       "Type": "String"
+    },
+    {
+      "Name": "RoomEvents",
+      "Type": "String"
+    },
+    {
+      "Name": "payload_object_room_name_s",
```

Comment:
Custom table schema having column names like column_x. Custom table should have column with final field names.